docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_buzzer_control:
+
+********************
+Buzzer Controller
+********************
+
+What is the Buzzer
+====================
+Your module is capable of inducing specific vibration frequencies in order to create audible tones. These buzzer tones are used to indicate several module statuses. 
+Here, we will discuss all possible “songs” your module may play during normal operation and their meanings. We will also discuss how to use the 
+:ref:`buzzer control client <buzzer_control>` in order to create and play your own songs.
```

Comment:
I'm not really sure what our typical standard is, but I kind of feel like Buzzer Control should be capitalized